### ISSUE TITLE
Snomed concept_relationship = 'Concept replaced by' not in Snomed concept_relationship = 'maps to'

### ISSUE THREAD

**Issue by mseal-cherokeehealth** [OHDSI/Vocabulary-v5.0#841]
There are snomed codes that are updated via 'concept replaced by' that are not under the concept_relationship 'maps_to

SQL proof ( as I do not want to import a list of 1,442 id's):

```
SELECT  x.concept_id, c.concept_id
FROM saftinet.dbo.concept (NOLOCK) x
INNER JOIN saftinet.dbo.concept_relationship (NOLOCK) cr
 ON   x.concept_id = cr.concept_id_1 AND cr.relationship_id = 'Concept replaced by'
INNER JOIN saftinet.dbo.concept (NOLOCK) c
 ON cr.concept_id_2 = c.concept_id
WHERE x.vocabulary_id = 'SNOMED' AND c.vocabulary_id = 'SNOMED'
  AND x.standard_concept <> 'S' AND c.standard_concept = 'S'

EXCEPT

SELECT  x.concept_id, c.concept_id
FROM saftinet.dbo.concept (NOLOCK) x
INNER JOIN saftinet.dbo.concept_relationship (NOLOCK) cr
 ON   x.concept_id = cr.concept_id_1 AND cr.relationship_id = 'maps to'
INNER JOIN saftinet.dbo.concept (NOLOCK) c
 ON cr.concept_id_2 = c.concept_id
WHERE x.vocabulary_id = 'SNOMED' AND c.vocabulary_id = 'SNOMED'
  AND x.standard_concept <> 'S' AND c.standard_concept = 'S'
```

**Comment by RogerJCarlson** [comment]
Here are a couple of threads I created in the Forums which may be related
[Anatomic Sites mapped to Observations in latest vocabulary](https://forums.ohdsi.org/t/anatomic-sites-mapped-to-observations-in-latest-vocabulary/19174)
[Device_concept_ids invalidated with no ‘Maps To’ or ‘Concept Replaced By’](https://forums.ohdsi.org/t/device-concept-ids-invalidated-with-no-maps-to-or-concept-replaced-by/19175)

**Comment by TinyRickC137** [comment]
Yes, not all of them are there. Thank you for reporting, it is an issue and we are aware of it. 
There may be a lot of reasons why these relationships are missing. To avoid that in the future, we created a specific [QA process](https://github.com/OHDSI/Vocabulary-v5.0/blob/master/working/manual_checks_after_generic_update.sql#L917), but old ones are still there.

@RogerJCarlson This has nothing to do with forum posts, I replied to both of them.
